src/mocks/msw/handlers.ts

```diff
@@ -1,18 +1,35 @@
 import { rest } from 'msw'
 import { SynapseError } from '../../lib/utils/SynapseClient'
-import { entityHandlers } from './handlers/entityHandlers'
-import { userProfileHandlers } from './handlers/userProfileHandlers'
+import { entityHandlers, getEntityHandlers } from './handlers/entityHandlers'
+import {
+  getUserProfileHandlers,
+  userProfileHandlers,
+} from './handlers/userProfileHandlers'
+import {
+  BackendDestinationEnum,
+  getEndpoint,
+} from '../../lib/utils/functions/getEndpoint'
 
 // Simple utility type that just indicates that the response body could be an error like the Synapse backend may send.
 export type SynapseApiResponse<T> = T | SynapseError
 
-const handlers = [
+const getHandlers = (backendOrigin: string) => [
```

Comment:
Just parameterizing the handlers so we can get handlers for a particular stack (e.g. the mock stack)